wellregistry/registry/migrations/0001_create_db_users.py

```diff
@@ -0,0 +1,176 @@
+"""
+Migrate initial users and roles.
+https://docs.djangoproject.com/en/3.0/topics/migrations/
+
+This one migration should be run on the 'postgres' database alias.
+> python manage.py migrate --database=postgres  registry 0000
+All subsequent migrations should be run on the 'migration'
+> python manage.py migrate --database=migration
+
+"""
+from django.db import migrations
+from wellregistry.settings import DATABASE_USERNAME
```

Comment:
You could also use ```from django.conf import settings```.  It might arguably be better to use the django settings database settings dictionary DATABASES.

Reply:
This is interesting. I will check it out.

Reply:
I got this to work cleanly. Thanks for the suggestion. While I saw this feature I did not expect my env vars to be in there.